config/virtuoso/virtuoso-production.ini

```diff
@@ -95,11 +95,11 @@ TransactionAfterImageLimit = 150000000	; max 150MB transaction, total transient
 ;NumberOfBuffers          = 340000
 ;MaxDirtyBuffers          = 250000
 ;; Uncomment next two lines if there is 8 GB system memory free
-; NumberOfBuffers          = 680000
```

Comment:
servers are not that big, changing this to give them a little more breathing room (our data isn't big either so should be fine)